src/registry.cpp

```diff
@@ -36,6 +36,7 @@
 #endif
 
 std::string Registry::ReadStrValue(HKEY hkey, std::string const& key, std::string const& val, REGVIEW view) {
+#ifndef _ARM_
```

Comment:
Besides that it's a bit hard to get RTP entries in the registry without a ARM installer :D